react/controls/FullscreenButton.tsx

```diff
@@ -0,0 +1,25 @@
+import React, { FunctionComponent } from 'react'
+import { Icon } from 'vtex.store-icons'
+
+import styles from '../styles/styles.css'
+
+export interface FullscreenButtonProps {
+  toggleFullscreenMode: () => void
+  cssHandles: any
+}
+
+const FullscreenButton: FunctionComponent<FullscreenButtonProps> = ({
+  toggleFullscreenMode,
+  cssHandles,
+}) => {
+  return (
+    <button
+      className={`${cssHandles.fullscreenButton} ${styles.button} ml4 absolute bottom-2 right-0`}
```

Comment:
@klzns considering the feedback that we had about our default styles, do you think it makes sense to add this tachyons here instead of adding this style inside the `store-theme` app?